tests/test_numbers.py

```diff
@@ -1,66 +1,97 @@
 """Tests for all number generators."""
 
+import string
 import sys
+from collections import namedtuple
+from functools import partial
 
+from numpy.core.numeric import base_repr
 import pytest
 
 from fauxfactory import (
+    gen_hexadecimal,
     gen_integer,
+    gen_number,
+    gen_octagonal,
     gen_negative_integer,
     gen_positive_integer,
 )
 
 
-def test_gen_integer_1():
-    """Create a random integer with no range limits."""
-    integer_types = (int,)
-    result = gen_integer()
-    assert isinstance(result, integer_types)
+DIGITS = string.digits + string.ascii_lowercase
```

Comment:
If you create a constant (see my previous comment) then we could import it here :)